dockers/docker-sonic-telemetry/telemetry.sh

```diff
@@ -76,4 +76,12 @@ else
     TELEMETRY_ARGS+=" --threshold 100"
 fi
 
+IDLE_CONN_DURATION=$(echo $GNMI | jq -r '.idle_conn_duration')
+if [[ $IDLE_CONN_DURATION =~ ^[0-9]+$ ]]; then
+    TELEMETRY_ARGS+=" --idle_conn_duration $IDLE_CONN_DURATION"
+else
```

Comment:
If invalid value, is it better to report error and exit?

Reply:
for missing case, it is okay to use default value.

Reply:
Done, did the same for threshold field as well.